nighthawk/source/client/output_formatter_impl.cc

```diff
@@ -0,0 +1,86 @@
+#include "nighthawk/source/client/output_formatter_impl.h"
+
+#include <chrono>
+#include <sstream>
+
+#include "common/protobuf/utility.h"
+
+namespace Nighthawk {
+namespace Client {
+
+OutputFormatterImpl::OutputFormatterImpl(Envoy::TimeSource& time_source, const Options& options,
+                                         const std::vector<StatisticPtr>& merged_statistics,
+                                         const std::map<std::string, uint64_t>& merged_counters)
+    : time_source_(time_source), options_(options), merged_statistics_(merged_statistics),
+      merged_counters_(merged_counters) {}
+
+ConsoleOutputFormatterImpl::ConsoleOutputFormatterImpl(
+    Envoy::TimeSource& time_source, const Options& options,
+    const std::vector<StatisticPtr>& merged_statistics,
+    const std::map<std::string, uint64_t>& merged_counters)
+    : OutputFormatterImpl(time_source, options, merged_statistics, merged_counters) {}
+
+std::string ConsoleOutputFormatterImpl::toString() const {
+  std::stringstream s;
+
+  // TODO(oschaaf): echo non-default options to CLI output?
+  s << "Merged statistics:\n";
+  for (auto& statistic : merged_statistics_) {
+    if (statistic->count() > 0) {
+      s << fmt::format("{}: {}\n", statistic->id(), statistic->toString(), "\n");
```

Comment:
Nit: why is the final "\n" an argument rather than part of the format string?

Reply:
Nice catch, that was a remnant of earlier stuff, and shouldn't be there at all. Removed.